app.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics.controller");
+
+app.use(express.json());
```

Comment:
Too early to include this line - this allows express to parse incoming JSON on the request. It comes into play when we handle bodies on the request like a comment for example.
Since it's not needed at this point, it's ideal to get rid of it